galsim/scene.py

```diff
@@ -0,0 +1,412 @@
+        >>>>     im_real.write('im_real_'+str(ind)+'.fits')
+        >>>>     im_param.write('im_param_'+str(ind)+'.fits')
+
+    This code snippet will draw images of the first 10 objects in the COSMOS catalog, at slightly
+    lower resolution than in COSMOS, with a real image and its parametric representation for each of
+    those objects.
+
+    Initialization
+    --------------
+
+    @param file_name    The file containing the catalog. [default: None, which will look for the
+                        COSMOS catalog in $PREFIX/share/galsim.  It will raise an exception if the
+                        catalog is not there telling you to run galsim_download_cosmos.]
+    @param use_real     Use realistic galaxies or parametric ones?  [default: True]
+    @param preload      Keyword that is only used for real galaxies, not parametric ones, to choose
+                        whether to preload the header information.  If `preload=True`, the bulk of  
+                        the I/O time is in the constructor.  If `preload=False`, there is
+                        approximately the same total I/O time (assuming you eventually use most of
+                        the image files referenced in the catalog), but it is spread over the
+                        various calls to getGal() and getPSF().  [default: False]
+    @param image_dir    Keyword that is only used for real galaxies, not parametric ones, to specify
+                        the directory of the image files.
+                        If a string containing no `/`, it is the relative path from the location of
+                        the catalog file to the directory containing the galaxy/PDF images.
+                        If a path (a string containing `/`), it is the full path to the directory
+                        containing the galaxy/PDF images. [default: None]
+    @param dir          The directory of catalog file. [default: None]
+    @param noise_dir    Keyword that is only used for real galaxies, not parametric ones.
+                        The directory of the noise files if different from the directory of the 
+                        image files.  [default: image_dir]
+    @param deep_sample  Modify fluxes and sizes of galaxies in order to roughly simulate an F814W<25
+                        sample? [default: False]
+    @param exclude_fail For catalogs of parametric galaxies, exclude those that have failures in the
+                        parametric fits?  [default: True]
+
+    Attributes
+    ----------
+
+    After construction, the following attributes are available:
+
+    nobjects     The number of objects in the catalog
+    obj_type     Either 'real' or 'parametric', depending on which way of representing galaxies was
+                 chosen.
+    deep         Either 'True' or 'False', depending on whether modifications of the object sizes
+                 and fluxes were used to simulate a deeper I<25 sample ('True') or not.
+
+    """
```

Comment:
Well, I discovered one problem with my proposed scheme.  I was planning to have the code automatically find the file with the fits by inserting '_fits' into the file name before '.fits'.  So they would just need to specify the file for the real galaxy catalog, and it would automatically be able to load both.

This works for when they don't actually specify a file name, since it can find the right file in the share/galsim/COSMOS_23.5_training_sample directory.  But it doesn't work for demo11, since the real and parametric  catalogs in the example/data directory don't match up.  We have real_galaxy_catalog_example.fits and real_galaxy_23.5_500_fits.fits, which correspond to different idents.  

Could we replace one of these so they do match up?  I would suggest replacing the parametric catalog with a real_galaxy_catalog_example_fits.fits to correspond to the real galaxy catalog we already had.  You only use 10 of the 500 parametric galaxies in demo13 after all, so 10 out of 100 is just as good as 10 out of 500.  And the user can still download the full catalogs to get thousands.  So it's probably fine to only distribute 100 in the repo.